clients/go/README.md

```diff
@@ -0,0 +1,51 @@
+# Myanmar Tools Golang Documentation
+
+This documentation is for Golang specific usage of *Myanmar Tools*.  For general documentation.
+
+## Installation
+```bash
+$ go get -u github.com/ayeminag/go-myanmartools
```

Comment:
can you update this url to point to github.com/google now?

Reply:
@sffc done bro